content/blogs/cbialorucki-tests-2.md

```diff
@@ -0,0 +1,65 @@
+---
+title:       "Progress update: fixing the ReactOS test suite"
+author:      "Carl Bialorucki"
+date:        2025-11-04
+tags:        [ "newsletter", "tests", "test-suite" ]
```

Comment:
I think it's a good idea to include a cover image, otherwise our homepage starts to look like a 404 page.

<img width="1075" height="698" alt="Image" src="https://github.com/user-attachments/assets/4f559c8a-9dcf-4f42-b592-74bc07cb822b" />